src/app/core/services/recently-used-items.service.ts

```
@@ -2,21 +2,36 @@
 })
 export class RecentlyUsedItemsService {
-  constructor(private apiService: ApiService, private projectsService: ProjectsService) {}
+  constructor(private SpenderPlatformV1ApiService: SpenderPlatformV1ApiService, private projectsService: ProjectsService) {}
+
+  formatRecentlyUsedFields(data: RecentlyUsed): RecentlyUsed {
+    const idFields = ['project_ids', 'category_ids', 'cost_center_ids'] as const;
+  
+    return idFields.reduce((acc, field) => {
+      const value = data[field];
+      return {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)